### ISSUE TITLE
Allow setting a manuever node delta V by its raw KSP vector.

### ISSUE THREAD

**Issue by Dunbaratu** [KSP-KOS/KOS#1093]
Right now NODE:DELTAV is a GET-only suffix, which is annoying when you're trying to operate in SHIPRAW coordinates rather than navball coordinates.  In raw coordinates, if you have a deltav burn vector, you can't just plug it directly into the maneuver node to make a node.  You have to first convert it into whatever its navball coordiantes are, not NOW, but what they will be at the point of the burn in the future after the navball has rotated.  It's a lot of infuriating work just to get around the fact that you can't do:
```
set mynode:DELTAV to myvector.
```
:DELTAV, and :BURNVECTOR should be settable, and there should be a second constructor variant that lets you construct the node by plugging in the deltav vector directly.

**Comment by Dunbaratu** [comment]
This is significantly more complex that it seemed at first.  The problem is that the Manuever Node class in KSP *itself* lacks a means of setting it by plain unity Vector3, or KSP Vector3d.  It imposes the need to set it by coordinates in prograde/radial/normal reference frame, rather than in XYZ reference frame. 

The twiddling knobs you see in the user interface are *actually* mapped 1-to-1 to how the data is really stored under the hood.  You can't store an XYZ raw vector in a manuever node.

Which means that to implement this requires that our code would read the XYZ vector from the user, predict what the ship's future position and velocity will be at future time T where the node is placed, but express those in the current XYZ frame rotation.  Then it can use that to work out how to convert that vector into the prograde/radial/normal reference frame at THAT future time, and store the values that way.

And they'd become wrong the moment you move the time of the node to a new location of the orbit - the node's deltaV would no longer point the same way it did before in XYZ terms when it was stored for a different point in time..

Therefore this issue is way more complex than originally thought.

Given that KOS has a Node class to wrap around the KSP class, maybe kOS could store the raw vector outside the KSP node, and then somehow apply it to the KSP node later?  This could get messy.

**Comment by Dunbaratu** [comment]
I'm going to close this as a "wontfix" because the inherent limits in the KSP code itself would mean we'd have to do a LOT of mapping back in and out of the script's way of thinking to implement it.  For example, if I set a maneuver node by a Ship-Raw vector, and then AFTER that I slide the node to a new timestamp a quarter of an orbit later 90 degrees around the planet, does that mean I intend to rotate that raw vector 90 degrees to match the ship's predicted velocity at that new time, or do I intend the deltaV vector to stay pointed the way it was?  If the Ship-Raw vector was roughly prograde-ish when I created the manuever node, and then slid it a quarter period ahead, it's now pointing roughly straight up away from the planet in this new position.  Was that the intention?  Or was the act of using a raw vector supposed to mean it stays that way?  If it was supposed to stay that way it means we need to remember this in some wrapper information around KSP's native maneuver node, which does not act like this, so we can constantly map in and out of its way of thinking.

Given that it could be either one, it's probably better to force the script writer to explicitly decide and make them perform the mapping by hand into and out of the node's prograde/radial/normal viewpoint themselves, and deal with the fact that those directions differ at different predicted timestamps in the future.  It's a bit ugly, but at least then we aren't deciding for them and getting it wrong.
